test/check-application

```diff
@@ -384,6 +401,12 @@ class TestApplication(testlib.MachineCase):
         self.assertIn('8001 8001 tcp', ports)
         self.assertNotIn('7001 7001 tcp', ports)
 
+        env = m.execute("podman inspect --format '{{.Config.Env}}' busybox-with-tty");
```

Comment:
This is fine. Just as a thing to consider, to test this all the way through, this could also do `podman exec busybox-with-tty env`, and skip making assumptions about the `podman inspect` API.